client/index.tsx

```diff
@@ -10,16 +11,33 @@ import { AuthProvider } from './auth/AuthProvider';
 import RequireAuth from './components/RequireAuth';
 import App from './App';
 
+const httpLink = createHttpLink({
```

Comment:
Just a note to briefly go through this fn

Reply:
We reviewed this, are we ok now?

Reply:
All set on my end

Reply:
Yup!